charts/cockroachdb/19.1.5/questions.yml

```diff
@@ -0,0 +1,15 @@
+--- 
+categories: 
+  - Config
+default: 100Gi
+description: "Size of volume for each CockroachDB Node/Pod"
+group: Config
+label: "Storage per Node/Pod"
+labels: 
+  - 
+    io.rancher.certified: partner
+questions: 
+  - 
```

Comment:
these formats are incorrect, can u please refer to the existing chart as an example to fix them, thanks.

Reply:
hopefully this latest format change is what you're looking for